codec-http2/src/main/java/io/netty/handler/codec/http2/CompressorHttp2ConnectionEncoder.java

```diff
@@ -256,7 +257,7 @@ private void initCompressor(int streamId, Http2Headers headers, boolean endOfStr
      * @param stream The stream for which {@code compressor} is the compressor for
      * @param compressor The compressor for {@code stream}
      */
-    private static void cleanup(Http2Stream stream, EmbeddedChannel compressor) {
+    private void cleanup(Http2Stream stream, EmbeddedChannel compressor) {
```

Comment:
This is used in inner class (not that you introduced this).  Make package protected.